LSE-61.tex

```diff
@@ -3684,7 +4465,8 @@ \subsection{Timely Publication of Level 2 Data Releases}
 \label{DMS-REQ-0006}
 \textbf{ID:} DMS-REQ-0006 (Priority: 1b)
 
-\textbf{Specification:} The DMS shall make available to end-users all Level-2 data products as soon as possible following the execution of the Data Release Production and the verification of science data quality.
+\textbf{Specification:}
+The DMS shall make available to end-users all Level-2 data products as soon as possible following the execution of the Data Release Production and the verification of science data quality.
```

Comment:
There is still some project-level sense behind this requirement, but it doesn't make sense as a _DM_ requirement.  At Project level, in the context of LPM-261, it means "all Full Users get access at the same time, with the associated publication rights, and no staff members receive publication rights before this time, regardless of their role in production or pre-release data quality assessment".

Reply:
This is in the *deleted* requirements section of the document. It was a formatting change.

Reply:
I missed your reply.  My comment was a response to it being a deleted requirement.  I only happened to _notice_ it because of the formatting change.  So in some formal sense it is a comment unrelated to this particular PR.